proto/iotics/api/input.proto

```diff
@@ -33,11 +33,13 @@ service InputAPI {
 }
 
 // Representation of an input.
-message Input {
-  // Input identifier (unique within the scope of a twin identifier's input set)
-  InputID id = 1;
-  // Twin unique identifier (twin to which the input belongs)
-  TwinID twinId = 2;
+message InputID {
```

Comment:
Should this be used for both reads and creates? For create, the host_id parameter makes no sense.

Reply:
Now 
the "full identifier of a twin is" => host twin did  + twin did 
the "full identifier of a feed is" => host twin did  + twin did + feed id
the "full identifier of an input is" => host twin did  + twin did + input id

For any local actions like describe, create, ... passing the  full identifier is not mandatory
For the local (only) actions like create, update, ... if you try to apply to a remote twin it will properly fail

That way you can reuse the ids (twinId, feedId, inputId) from one payload to an other without having to edit them.
For example search for some inputs then update them.